File: AgqrPlayer4Tv/AgqrPlayer4Tv/Activities/ErrorActivity.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using AgqrPlayer4Tv.Components.Fragments;
using Android.App;
using Android.Content;
using Android.OS;
using Android.Runtime;
using Android.Views;
using Android.Widget;

namespace AgqrPlayer4Tv.Activities
{
    [Activity(Label = "ErrorActivity")]
    public class ErrorActivity : Activity
    {
        protected override void OnCreate(Bundle bundle)
        {
            SetContentView(Resource.Layout.Main);

            if (bundle == null)
            {
                var fragment = new ConnectionErrorFragment();
                var transaction = FragmentManager.BeginTransaction();
                transaction.Add(Resource.Id.MainFrame, fragment);
                transaction.Commit();
            }

            base.OnCreate(bundle);
        }
    }
}

File: AgqrPlayer4Tv/app/src/main/java/org/misuzilla/agqrplayer4tv/component/activity/ErrorActivity.kt
package org.misuzilla.agqrplayer4tv.component.activity

import android.os.Bundle
import android.support.v4.app.FragmentActivity
import org.misuzilla.agqrplayer4tv.R
import org.misuzilla.agqrplayer4tv.component.fragment.ConnectionErrorFragment

class ErrorActivity : FragmentActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        if (savedInstanceState == null) {
            val fragment = ConnectionErrorFragment()
            val transaction = supportFragmentManager.beginTransaction()
            transaction.add(R.id.main_frame, fragment)
            transaction.commit()
        }
    }
}
